addon/server.js

```diff
@@ -62,7 +62,7 @@ function createPretender(server) {
          mirage/config.js file. Did you forget to add your namespace?`
       );
     };
-  });
+  }, { trackRequests: false });
```

Comment:
no effect to use I think, originally solves this https://github.com/samselikoff/ember-cli-mirage/issues/509